Power of Thor bas824.cpp

```diff
@@ -1,24 +1,25 @@
 // # The Power of Thor: Literate Code
```

Comment:
There are two files -- this one (in the wrong directory, but I think with your edits) and another file which is in the correct directory, but doesn't have edits. Please fix.